src/index.js

```diff
@@ -38,3 +37,19 @@ export const renderReactWithAphrodite = (name, component) => hypernova({
     return component;
   },
 });
+
+export const renderReactWithAphroditeStatic = (name, component) => hypernova({
+  server() {
+    return (props) => {
+      const { html, css } = StyleSheetServer.renderStatic(() => (
+        ReactDOMServer.renderToStaticMarkup(React.createElement(component, props))
+      ));
+
+      const style = `<style data-aphrodite>${css.content}</style>`;
+
+      return `${style}\n${html}`;
```

Comment:
@goatslacker Does using `html` here directly seem okay, as opposed to using the `markup` returned from `serialize`? I would assume so, but want to make sure there isn't a security hole in not using `serialize`, for instance.